airflow/operators/sensors.py

```diff
@@ -331,3 +331,44 @@ def poke(self, context):
         logging.info(
             'Checking if the time ({0}) has come'.format(self.target_time))
         return datetime.now().time() > self.target_time
+
+class HttpSensor(BaseSensorOperator):
+    """
+    Executes an HTTP get statement until the specified criteria is met.
+
+    :param conn_id: The connection to run the sensor against
+    :type conn_id: string
+    :param url: To pass, the url must respond with 
+    """
+
+    @apply_defaults
+    def __init__(self, conn_id, url, params={}, headers=None, extra_options={}, *args, **kwargs):
+        super(HttpSensor, self).__init__(*args, **kwargs)
+
+        self.url = url
+        self.conn_id = conn_id
+        self.params = params
+        self.headers = headers
+        self.extra_options = extra_options
+
+        session = settings.Session()
+        site = session.query(DB).filter(DB.conn_id == conn_id).first()
+        if not site:
+            raise AirflowException("conn_id doesn't exist in the repository")
+        self.conn_id = conn_id
+        self.hook = hooks.HttpHook( method='GET', http_conn_id=self.conn_id )
+        session.commit()
+        session.close()
+
+    def poke(self, regex=None):
+        logging.info('Poking: ' + self.url)
+        result, content = self.hook.run( self.url, data=self.params, headers=self.headers, extra_options=self.extra_options )
+        if not result:
+            return False
+
+        if regex != None:
+            # run regex for fail condition
+            pass
+
+        return result
```

Comment:
BaseSensorOperator expects poke to return `True` or `False`. Though result here would eval to `True`